family_context/settings.py

```diff
@@ -38,6 +39,48 @@
     default="https://127.0.0.1",
 )
 
+LOG_LEVEL = config("LOG_LEVEL", default="INFO")
+LOG_DESTINATION = config("LOG_DESTINATION", default="file_output")
+choose_log_handler = {"file_output": "file", "console_output": "console"}
+
+# By default, log to console
+LOGGING = {
```

Comment:
Something in this configuration is stopping the middleware logs to work properly - if you replace it with the default examples it works.

https://docs.djangoproject.com/en/4.2/topics/logging/#examples

Reply:
It wasn't working earlier because since the if LOG_DESTINATION block was always true, as you said. so it replaced the console handler with the file handler. The new commit changes that so that new handlers are appended to the existing list of handlers instead of replacing them.